### PR TITLE
Created FDMI high level decomposition .md page

### PR DESCRIPTION
**Pull request by swatid-seagate**
### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [ ] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [ ] tested locally
 - [ ] added new dependencies
 - [ ] updated the docs
 - [ ] added a test

-----
[View rendered Community_Roadmap.md](https://github.com/swatid-seagate/cortx/blob/main/Community_Roadmap.md)
[View rendered QUICK_START.md](https://github.com/swatid-seagate/cortx/blob/main/QUICK_START.md)
[View rendered doc/FDMI-High-Level-Decomposition.md](https://github.com/swatid-seagate/cortx/blob/main/doc/FDMI-High-Level-Decomposition.md)
[View rendered doc/Xyratex-FDMI-Design-Notes.md](https://github.com/swatid-seagate/cortx/blob/main/doc/Xyratex-FDMI-Design-Notes.md)
[View rendered doc/Xyratex_FDMI_HLD.md](https://github.com/swatid-seagate/cortx/blob/main/doc/Xyratex_FDMI_HLD.md)
[View rendered terms_and_conditions.md](https://github.com/swatid-seagate/cortx/blob/main/terms_and_conditions.md)

### CODE REVIEW

**Comment by huanghua78** [comment]
Please rename all `xyratex` to `seagate`

File: Community_Roadmap.md [hunk 1]
```diff
@@ -47,7 +47,11 @@
             <td></td>
             <td> </td>
             <td> <img src="https://www.countryflags.io/ie/flat/64.png"> </td>
-            <td> </td>
```
Comment (johnbent) [inline Community_Roadmap.md:50]: This should not be in a PR about FDMI.

File: QUICK_START.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 This document provides quick access to several different ways to set up CORTX for installation, testing, running, or deployment. CORTX is available in multiple formats, including as a pre-built virtual machine image - the easiest and fastest way to get CORTX up and running. Alternatively, CORTX can be built into a binary format from source code.  Finally, for those interested in running different configurations of CORTX, there are a variety of different options available. These include everything from the full installation to more slimmed down installations, such as configuring and running motr, the core object storage layer, on its own.  We have attempted to create easy-to-follow instructions for all of these installation/test methods which are organized below in order of difficulty.
  
  1. Download a CORTX virtual machine image to get up and running in minutes
-    1. On a single node: [LINK](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst)
+    1. On a single node: [LINK](doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst)
```
Comment (johnbent) [inline QUICK_START.md:6]: This change should not be in a PR about FDMI.

File: QUICK_START.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node: TODO
     1. Across a cluster: [requires verification](doc/scaleout/README.rst)
 5. Build entire CORTX into OVA (virtual machine image): TODO
-    1. Run it on a single node: TODO 
+    1. Run it on a single node: [LINK](https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst)
```
Comment (johnbent) [inline QUICK_START.md:24]: This change should not be in a PR about FDMI.

**Review by johnbent — CHANGES_REQUESTED** [review]
Hey Swati. Good job on making some initial FDMI documents! However, FDMI documents should be in the motr repo. @lia54 can help since she is also working on FDMI documentation.

**Comment by r-wambui** [comment]
@swatid-seagate For git workflow, it's best to create a PR related to each task/feature/bug, etc. The PR title you have currently is FDMI, so any file changes should only reflect anything about FDMI; however, you have other changes on the PR that relate to the quick start guide and not fdmi.
Could you please create multiple PRs where each relates to a given task e.g

1. for fdmi
2. for community roadmap
3. for quick start guide.

Doing this makes reviewing easier. You could also do the same for the PR in motr repo.

**Comment by r-wambui** [comment]
@swatid-seagate Could you please close this pull request(PR), if you have already transferred the changes to motr repo and the PR is no longer required.

**Pull request closed without merging**